### PR TITLE
review security warnings

### PR DESCRIPTION
**Pull request by k41n**

### CODE REVIEW

File: app/controllers/feeds_controller.rb [hunk 1]
```diff
@@ -193,7 +196,7 @@ def ical(mode = :personal)
       else
         event.start = to_localtime(tz, m.due_at).beginning_of_day + 8.hours
       end
-      event.duration = "PT#{480}M"
+      event.duration = 'PT480M'
```
Comment (ari) [inline app/controllers/feeds_controller.rb]: Does this magic string need an explanation?

File: app/models/worklog_report.rb [hunk 2]
```diff
@@ -327,7 +327,7 @@ def key_from_worklog(w, r)
       "6_approved"
     elsif (property = Property.find_by_filter_name(current_user.company, r))
       w.task.property_value(property)
-    elsif r and (match = r.match(/ca_(\d+)/))
+    elsif r and (r.match(/ca_(\d+)/))
```
Comment (ari) [inline app/models/worklog_report.rb]: Do we know what this magic regex does?

File: app/controllers/feeds_controller.rb (hunk 1 above)
Comment (k41n) [inline app/controllers/feeds_controller.rb]: In this case, interpolation within the string with numbers is not needed

File: app/models/worklog_report.rb (hunk 2 above)
Comment (k41n) [inline app/models/worklog_report.rb]: Check for custom attributes id and return string with ca_id (custom attribute id)

File: app/controllers/feeds_controller.rb (hunk 1 above)
Comment (ari) [inline app/controllers/feeds_controller.rb]: actually I was still confused what PT480M means... (480 minutes?)

File: app/models/worklog_report.rb (hunk 2 above)
Comment (ari) [inline app/models/worklog_report.rb]: Bit of an ugly way to write custom SQL I guess.

**Comment by ari** [comment]
@k41n can you fix the test failures please.

**Comment by k41n** [comment]
@ari Tests were fixed.

**Pull request merged**